app/tasks.py

```diff
@@ -379,3 +380,65 @@ def make_query(self):
     def update_counter(self, counter, person):
         person.update_index(['old', 'new'])
         person.put()
+
+
+class NotifyBadReviewStatus(utils.BaseHandler):
+    """This task sends email notification when the number of unreviewed notes
+    exceeds threshold.
+    """
+    ACTION = 'tasks/notify_bad_review_status'
+
+    def task_name(self):
+        return 'notify-bad-review-status'
+
+    def get(self):
+        if self.repo:
+            try:
+                count_of_unreviewed_notes = \
+                    model.Note.unreviewed_record_count(self.repo)
+            except runtime.DeadlineExceededError:
+                logging.info("DeadlineExceededError occurs")
+                self.add_task_for_repo(
+                    self.repo, self.task_name(), self.ACTION)
+            except datastore_errors.Timeout:
+                logging.info("Timeout occurs in datastore")
+                self.add_task_for_repo(
+                    self.repo, self.task_name(), self.ACTION)
+            finally:
+                self._notify(count_of_unreviewed_notes)
+
+    def _notify(self, count_of_unreviewed_notes):
+        # TODO: Response should be modified
+        if self._is_ok_to_notify(count_of_unreviewed_notes):
+            self.send_mail(self._get_admin_user_email(),
+                           subject="PersonFinder: Please review your notes",
+                           body="{0} notes are not reviewed".format(
+                               count_of_unreviewed_notes))
+            return True
+        else:
+            return False
+
+    def _is_ok_to_notify(self, count_of_unreviewed_notes):
+        email = self._get_admin_user_email()
+        if not email:
+            return False
+
+        thres = self._get_thres_unreviewed_notes()
+        if count_of_unreviewed_notes >= thres:
+            return True
+        else:
+            return False
+
+    def _get_admin_user_email(self):
+        return self.config.get('admin_user_email')
+
+    def _get_thres_unreviewed_notes(self):
+        try:
+            return int(self.config.get('thres_unreviewed_notes'))
```

Comment:
self.config.get('thres_unreviewed_notes') should be already int, so you don't need int().

You may still want to assert that its type is int. It would be debatable what we should do when the assertion fails. But I would prefer to fail the request (e.g., by using assert statement) instead of silently falling back to the default value. Because it would be easier to notice the mistake.